src/feature/control/control_getinfo.c

```diff
@@ -325,6 +325,51 @@ getinfo_helper_current_time(control_connection_t *control_conn,
   return 0;
 }
 
+/**
+ * Switch between microdesc vs networkstatus descriptor dumps
+ * Assumes that question would be either:
+ * - dir/status-vote/current/consensus-microdesc
+ * or
+ * - dir/status-vote/current/consensus
+ */
+STATIC int
+getinfo_helper_current_consensus(const char* question,
+                          char** answer,
+                          const char** errmsg)
+{
+  // Ensures question contains a request for
+  // ns or microdesc consensus
+  if (
```

Comment:
Why is there a gap here? You could put the `strcmp` in the `if` line, except if you have something specific in mind.